orchestrators/FinanceOperations/tests/Unit/Rules/BudgetAvailableRuleTest.php

```
@@ -4,474 +4,131 @@
                 };
             }
-            public function getAvailableAmount(string $tenantId, string $budgetId, ?string $costCenterId = null): string {
-                return '10000.00';
-            }
-        });
 
-        $context = new class {
-            public function getTenantId(): string {
-                return 'tenant-001';
+            public function getAvailableAmount(
+                string $tenantId,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)